src/main/java/com/feature/toggle/simple/SimpleToggleFinder.java

```diff
@@ -11,8 +12,12 @@ public class SimpleToggleFinder {
 
     @Transactional(readOnly = true)
     public boolean isActive(String toggleId) {
-        return simpleToggleRepository.findByToggleId(toggleId)
-                .orElseThrow(() -> new RuntimeException("유효하지 않은 토글입니다."))
-                .isUseYn();
+        SimpleToggle toggle = simpleToggleRepository.findByToggleId(toggleId);
+
+        if(ObjectUtils.isEmpty(toggle)) {
+            return false;
+        }
+
+        return toggle.isUseYn();
```

Comment:
* Optional을 활용하면 아래처럼 활용 가능합니다.
```suggestion
       return simpleToggleRepository.findByToggleId(toggleId)
                       .map(SimpleToggle::isUseYn)
                       .orElse(false);
```

* 가독성이 별로라면 
```suggestion
       Optional<SimpleToggle> toggle = simpleToggleRepository.findByToggleId(toggleId);

        if (toggle.isPresent()) {
            return toggle.get().isUseYn();
        }
        
        return false;
```